projects/python-client/tests/client/e2e/test_apps.py

```diff
@@ -15,6 +15,7 @@
 # TODO: FIX TEST TO USE ENVIRONMENT VALUE FROM ENVIRONMENT OBJECT
 
 
+@pytest.mark.skip
```

Comment:
you can do this at the entire module level - see https://docs.pytest.org/en/latest/how-to/skipping.html#summary

```python
# at top of module before all tests
pytestmark = pytest.mark.skip("LeagacyApp tests disabled")
```